apps/test-pwm/src/main.cpp

```diff
@@ -0,0 +1,28 @@
+#include "mbed.h"
+
+PwmOut pwmLED(LED1);
+
+const auto k_period     = 1ms;
```

Comment:
could you explain why use const in one case but use constexpr in the other?

Reply:
good catch. i just missed it lol